resources/views/layouts/frontend_master.blade.php

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="vi">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>OSRU - {{--{{ $title }}--}}</title>
+    <link href="{{ asset('/favicon.ico') }}" rel="icon">
```

Comment:
```suggestion
    <link href="{{ asset('favicon.ico') }}" rel="icon">
```
các link còn lại tương tự nha anh